ui/src/app/job-list/table/bulk-edit/bulk-edit.component.ts

```diff
@@ -0,0 +1,56 @@
+import {Component, Inject} from "@angular/core";
+import {MAT_DIALOG_DATA, MatDialogRef} from "@angular/material";
+import {DisplayField} from "../../../shared/model/DisplayField";
+
+@Component({
+  selector: 'jm-bulk-edit-component',
+  templateUrl: 'bulk-edit.component.html',
+  styleUrls: ['bulk-edit.component.css']
+})
+export class JobsBulkEditComponent {
+  // currently Cromwell's limit; if there is some variability in other backends
+  // this should be moved to a config
+  public readonly labelCharLimit = 255;
+
+  constructor(
+    public dialogRef: MatDialogRef<JobsBulkEditComponent>,
+    @Inject(MAT_DIALOG_DATA) public data: any) { }
+
+  cancelChanges(): void {
+    this.dialogRef.close();
+  }
+
+  getFieldType(df: DisplayField): string {
+    return df.fieldType.toString();
+  }
+
+  getFieldOptions(df: DisplayField): string[] {
+    return df.validFieldValues;
+  }
+
+  getFieldDefaultValue(displayFieldItem: {
```

Comment:
Since you use this type in multiple places, probably makes sense to define this as an interface